conftest.py

```diff
@@ -0,0 +1,268 @@
+"""
+Pytest configuration for integration tests with VCR.
+
+This module configures VCR (Video Cassette Recorder) to record and replay
+HTTP interactions during tests. It automatically scrubs sensitive information
+from cassettes to prevent accidental credential leaks.
+
+VCR behavior controlled by VCR_MODE environment variable:
+- VCR_MODE=new_episodes (PRs): Records new interactions only, replays existing cassettes
+- VCR_MODE=all (push to main): Re-records ALL cassettes to verify API interface
+- VCR_MODE=once (local default): Records new cassettes if missing, replays existing
+- VCR_MODE=none (strict playback): Uses existing cassettes only, no real API calls
+
+All integration tests marked with @pytest.mark.integration automatically use VCR.
+"""
+
+import os
+import re
+import pytest
+import vcr as vcrpy
+from decouple import config
+
+
+# Sensitive headers to redact in VCR cassettes
+SENSITIVE_HEADERS = [
+    "authorization",
+    "x-api-key",
+    "api-key",
+    "apikey",
+    "x-auth-token",
+    "auth-token",
+    "cookie",
+    "set-cookie",
+]
+
+
+def scrub_sensitive_data(response):
```

Comment:
Interesting how much harder this is in the Python version of VCR than in Ruby. It generally would look something like:

```ruby
 c.filter_sensitive_data('<REDACTED>') { ENV["HUD_API_KEY"] }
```

It doesn't care where the string shows up and simply replaces it anywhere it's found.